simplified-books-database/src/main/java/org/nypl/simplified/books/book_database/DatabaseFormatHandleAudioBook.kt

```diff
@@ -386,12 +388,23 @@ internal class DatabaseFormatHandleAudioBook internal constructor(
     ): List<Bookmark.AudiobookBookmark> {
       val tree = objectMapper.readTree(fileBookmarks)
       val array = JSONParserUtilities.checkArray(null, tree)
-      return array.map { node ->
-        BookmarkJSON.deserializeAudiobookBookmarkFromJSON(
-          kind = BookmarkKind.BookmarkExplicit,
-          node = node
-        )
+
+      val bookmarks = arrayListOf<Bookmark.AudiobookBookmark>()
+
+      array.map { node ->
```

Comment:
Not a blocker, but `array.forEach` would be more appropriate here. I see you made that change in DatabaseFormatHandleEPUB, looks like you just missed it here.

Reply:
Yes, you're right! Fixed it.